test/e2e/cypress/e2e/clusters_overview.cy.js

```diff
@@ -165,4 +165,19 @@ context('Clusters Overview', () => {
       cy.contains(hanaCluster1.name).should('not.exist');
     });
   });
+
+  describe('Restoration', () => {
```

Comment:
The tests is totally dependant on the orther of tests. Here you assume that the previous test was already executed and worked.
I think we should have a `before` clause where we check if the host is there, and if it is, deregister it.

Other option that I would accept is to move this test to the `Deregistration` describe, as in essence, the restoration is part of the deregistration.